enterprise/cmd/frontend/internal/compute/streaming/stream.go

```diff
@@ -115,6 +115,13 @@ LOOP:
 		_ = eventWriter.Event("error", streamhttp.EventError{Message: err.Error()})
 		return
 	}
+
+	if err := ctx.Err(); err == context.DeadlineExceeded {
```

Comment:
very nit:
```suggestion
	if err := ctx.Err(); errors.Is(err, context.DeadlineExceeded) {
```

it really doesn't make a difference here, but I'd rather avoid having any examples in the codebase of comparing errors with `==`

Reply:
nice, I was looking for this code and couldn't remember it!